bigquery_etl/parse_metadata.py

```diff
@@ -29,7 +29,13 @@ def is_valid_label(label):
         63 characters, and cannot be empty. Values can be empty, and have
         a maximum length of 63 characters.
         """
-        return re.match(r"[\w\d_-]+", label) and len(label) <= 63
+        return (
+            len(label) > 0
+            and re.match(r"[\w\d_-]+", label)
+            and len(label) <= 63
+            and (label.isdigit() or label.islower())
```

Comment:
I don't understand this line. I would expect label could be a mix of digits and lowercase letters, but this seems to enforce that it's either all digits or all letters,

Reply:
The problem here I was running into was that `"123".islower() == False` while `"123a".islower() == True`. 
I guess using `label.lower() == label` covers these cases better though.